modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
+... Select *Deploy* to redeploy the application with the updated Mule runtime engine version.
+... Repeat this for each application running on this Runtime Fabric.
+.. Verify Runtime Fabric is healthy:
+... Using your web browser, navigate to the *Runtime Fabrics* tab in Anypoint Runtime Manager and verify that the Runtime Fabric reports an Active status.
+... Open a terminal to a Runtime Fabric controller node.
+... Run `sudo gravity status` and confirm the cluster status is healthy.
+... Keep your terminal open to use in later steps.
+
+.. Get the latest version of the Runtime Fabric installer package:
+... Using your web browser, navigate to Anypoint Runtime Manager and select the *Runtime Fabrics* tab to view a list of your Runtime Fabrics.
+... Select the *Downloads* link.
+... Select the *Copy Link* icon next to *Installer package* to copy the code snippetL, as shown in the following example:
++
 ```
 rtfctl appliance upgrade --url https://<upgrade-package-information>
 ```
+... Paste the URL in a text editor for reference. 
+.. Start the upgrade procedure:
+... Using your terminal open on a controller node, run the following command, substituting the value of the URL parameter with the *Installer package* URL: 
++
+----
+sudo ./rtfctl appliance upgrade --url <cluster-installer-url>
+----
++
+This command also supports referencing an installer package already downloaded on the node, with the `--file` parameter.
++
+The command outputs a confirmation indicating the upgrade is being performed in the background.
++
+.. Run `sudo gravity status` on a node and verify that the cluster status is “updating”.
+.. Follow the progress on the upgrade procedure:
+... Using your terminal open on a controller node, run the following command: 
++
+----
+sudo gravity plan
+----
++
+.. Confirm the upgrade has completed successfully:
+... Run `sudo gravity status` and verify that the cluster status transitioned from “updating” to “active”.
+.. If the Runtime Fabric cluster version was 1.0.x prior to upgrading, and an HTTP proxy is in use, run this command to apply the HTTP proxy settings: 
++
+----
+sudo ./rtfctl apply http-proxy --confirm existing
+----
+.. Perform the following step on *every node* to make sure system configurations are up to date:
 
-[IMPORTANT]
-To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, upgrade to 1.4.54 before upgrading to 1.5.
+... Open a terminal to your Runtime Fabric controller/worker node.
+... If needed, download the latest `rtfctl` command-line utility:
++
+----
+cd /opt/anypoint/runtimefabric
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
++
+... Change file permissions for the `rtfctl` binary: 
++
+----
+chmod +x rtfctl
+----
++
+... Run the `apply system-configurations` command in `rtfctl`:
++
+----
+sudo ./rtfctl apply system-configuration 
+----
+
+== Resume an Upgrade
```

Comment:
Needs to be clear this is only for appliance upgrades.

Reply:
Updated this - thank you!